pkg/fanal/analyzer/analyzer.go

```diff
@@ -92,7 +92,7 @@ const GroupBuiltin Group = "builtin"
 
 func RegisterAnalyzer(analyzer analyzer) {
 	if _, ok := analyzers[analyzer.Type()]; ok {
-		log.Logger.Fatalf("analyzer %s is registered twice", analyzer.Type())
+		log.Fatal("The analyzer is registered twice", log.String("type", string(analyzer.Type())))
```

Comment:
It's easier to read without an article (the) being present.
```suggestion
		log.Fatal("Analyzer is registered twice", log.String("type", string(analyzer.Type())))
```